content/connect/index.textile

```diff
@@ -0,0 +1,224 @@
+ably.connection.on('connected', () => {
+  console.log('Connected to Ably!');
+});
+
+bc[ruby]. ably = Ably::Realtime.new('{{API_KEY}}')
+ably.connection.on(:connected) do
+  puts "Connected to Ably!"
+end
+
+bc[java]. AblyRealtime ably = new AblyRealtime("{{API_KEY}}");
+ably.connection.on('connected', new ConnectionStateListener() {
+  @Override
+  public void onConnectionStateChanged(ConnectionStateChange change) {
+    System.out.println("Connected to Ably!");
+  }
+});
+
+bc[csharp]. AblyRealtime ably = new AblyRealtime("{{API_KEY}}");
+ably.Connection.On(ConnectionEvent.Connected, args => {
+  Console.WriteLine("Connected to Ably!");
+});
+
+bc[objc]. ARTRealtime *ably = [[ARTRealtime alloc] initWithKey:@"{{API_KEY}}"];
+[ably.connection on:ARTRealtimeConnectionEventConnected callback:^(ARTConnectionStateChange *change) {
+    NSLog(@"Connected to Ably!");
+}];
+
+bc[swift]. let realtime = ARTRealtime(key: "{{API_KEY}}")
+realtime.connection.on(.connected) { change in
+    print("Connected to Ably!")
+}
+
+bc[flutter]. final realtime = ably.Realtime(key: '{{API_KEY}}');
+final channel = realtime
+  .on(ably.ConnectionStateChange.connected)
+  .subscribe((ably.ConnectionStateChange stateChange) {
+    print('Connected to Ably!');
+  }
+);
+
+blang[javascript,nodejs].
+  If using the promises interface:
+
+  bc[javascript]. const ably = new Ably.Realtime.Promise('{{API_KEY}}');
+  await ably.connection.once("connected");
+  console.log('Connected to Ably!');
+
+  bc[nodejs]. const Ably = require('ably');
+  const ably = new Ably.Realtime.Promise('{{API_KEY}}');
+  await ably.connection.once("connected");
+  console.log('Connected to Ably!');
+
+
+<span lang="ruby">Note that all examples on this page assume you are running them within an EventMachine reactor. Find out more in our "Realtime usage documentation":/realtime/usage.</span>
+
+If you're not using the SDK's auto-connect feature you can also connect with "@connect()@":/api/realtime-sdk/connection#connect to manually connect unless the state is already @connected@ or @connecting@.
+
+h2(#monitor). Monitor connections to an app
+
+Connection monitoring allows you to view and manage the "states of connections":/connect/states to Ably, showing events for individual people connecting and disconnecting. The "developer console":/account/LINK-ME also shows these events.
+
+This feature is intended for debugging, so once the number of new connections exceeds the number of messages per second permitted by the lifecycle channel, new events will be dropped. This means if you want a definitive list of everyone using your app you'd be best using "token authentication":/auth/token to create your own 'auth server'.
+
+CAVEAT-INPUT
+
+  Browser security models ensure that each tab is effectively sandboxed from every other so it is not possible to share connections between tabs. As such, if a user has three tabs open, each with a connection to Ably, that will count as three connections.
```

Comment:
Suggest moving the object of this sentence around:

```suggestion
It isn't possible to share a connection between browser tabs. This is because browser security models ensure that each tab is effectively sandboxed from the others. If a user has three tabs open, each with a connection to Ably, then this will count as three separate connections.
```